src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Order.sql

```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE dbo.[Order]
+(
+	OrderId NVARCHAR(10) NOT NULL PRIMARY KEY,
+	[Description] NVARCHAR(50) NOT NULL,
+	OrganisationId UNIQUEIDENTIFIER NOT NULL,
+	StatusId INT NOT NULL,

```

Comment:
It might be easier to follow the relationships of tables if the column name on one table is the same as the associated field on another table e.g. PK OrderStatusId -> StatusId So in order to reduce context switching when reading this syntax, here we could keep both fields the same name i.e. OrderStatusId -> OrderStatusId.

Reply:
That's a really good suggestion, thanks 👍